package.json

```diff
@@ -45,6 +45,7 @@
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^23.4.2",
```

Comment:
Not sure we need to add this to dev deps?

Reply:
Without `babel-jest` I see this error:

```
  ● writes when --write is passed

    expect(jest.fn()).toBeCalled()
    
    Expected mock function to have been called.

      27 | 
      28 |   await writePackageJSON({ write: true });
    > 29 |   expect(writeJsonFile).toBeCalled();
      30 |   writeJsonFile.mockReset();
      31 | 
      32 |   await writeBabelRC("./fixtures/babelrc.json5", { write: true });
```
Node v10.9.0

Reply:
@isacjunior, which version of `node` are you on? I use node10 and I don't see that error when doing `npm run test`. If you are using node <  8, do `npm run build` and then run the tests.

Reply:
Node: v10.9.0

Reply:
Ok. So,
`jest --version` gives `v22.4.4`
`node --version` gives `v10.0.0`
`babel-jest` is not installed.

and when I do `npm run test` on the `master` branch, all the tests run successfully.

Reply:
Note: When I installed my dependencies with `npm` this errors above not occurred, but with `yarn` this error remains.